.buildkite/pipeline.yaml

```diff
@@ -1,34 +1,25 @@
-.test_linux: &test_linux
-  label: 'Test and package app on Linux'
-  agents:
-    production: "true"
-    platform: "linux"
-  timeout_in_minutes: 60
-  env:
-    DOCKER_IMAGE: 'gcr.io/opensourcecoin/radicle-upstream:0.2.1'
-    SHARED_MASTER_CACHE: true
-  command: '.buildkite/run.sh'
-  artifact_paths:
-    - "app/dist/*.AppImage"
-    - "app/dist/*.snap"
-    - "app/cypress/screenshots/**/*.png"
-
-.test_macos: &test_macos
-  label: 'Test and package app on macOS'
-  agents:
-    production: "false"
-    platform: "macos"
-  timeout_in_minutes: 60
-  command: '.buildkite/run.sh'
-  artifact_paths:
-    - "app/dist/*.dmg"
-    - "app/cypress/screenshots/**/*.png"
-
 steps:
-  - branches: master
-    concurrency: 1
-    concurrency_group: master
-    <<: *test_linux
-    <<: *test_macos
-  - branches: "!master"
-    <<: *test_linux
+  - label: 'Test and package app on Linux'
+    command: '.buildkite/run.sh'
+    agents:
+      production: "true"
+      platform: "linux"
+    timeout_in_minutes: 60
+    env:
+      DOCKER_IMAGE: 'gcr.io/opensourcecoin/radicle-upstream:0.2.1'
+      SHARED_MASTER_CACHE: true
+    artifact_paths:
+      - "app/dist/*.AppImage"
+      - "app/dist/*.snap"
+      - "app/cypress/screenshots/**/*.png"
+
+  - label: 'Test and package app on macOS'
+    if: build.branch == 'master'
+    command: '.buildkite/run.sh'
+    agents:
+      production: "false"
```

Comment:
I left `mac-n-cheese` out of production because the "buildkite-agent pipeline upload" step which is required by all our pipelines runs on any production agent and it broke on the mac.

https://buildkite.com/monadic/radicle-registry/builds/1109#21d1936f-5353-4edf-9757-01c5af9e624a

Besides we haven't set up any filesystem isolation on the mini which is another reason I don't want to run any builds by untrusted branches on it.